internal/tools/orchestrator/dbclient/tmc_instance.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbclient
+
+import (
+	"time"
+
+	"github.com/jinzhu/gorm"
```

Comment:
you'd better use `https://github.com/go-gorm/gorm`

Reply:
no need to change. in DbClient use `jinzhu/gorm` , here only `err` is judged.